build-new-version.sh

```diff
@@ -93,7 +100,11 @@ cargo_filename() {
 }
 
 download_files() {
-    download "https://static.rust-lang.org/dist/$CHANNEL_FILE"
+ if [ x"$TARGET_VERSION" == x"nightly" -a "$NIGHTLY_DATE" ]; then
```

Comment:
Minor, but indentation here looks like it got off a bit.

Reply:
Oops, thanks